src/creatures/players/player.cpp

```diff
@@ -121,7 +123,7 @@ std::string Player::getDescription(int32_t lookDistance) {
 	capitalizeWords(subjectPronoun);
 
 	if (lookDistance == -1) {
-		s << "yourself.";
+		s << "yourself" << (title()->getCurrentTitle() == 0 ? "" : (", " + title()->getCurrentTitleName())) << ".";
```

Comment:
It might be interesting to turn this into a variable and reuse it in both places to avoid code duplication.